config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/raffaelespazzoli/volume-expander-operator
```

Comment:
This should reference `redhat-cop`?

Reply:
it is actually irrelevant, but I restored some more sensible values, here and in another point.